gc/verbose/VerboseWriterFileLoggingBuffered.cpp

```diff
@@ -94,7 +94,10 @@ MM_VerboseWriterFileLoggingBuffered::openFile(MM_EnvironmentBase *env, bool prin
 		return false;
 	}
 	
-	_logFileStream = omrfilestream_open(filenameToOpen, EsOpenWrite | EsOpenCreate | EsOpenTruncate, 0666);
+	int32_t openFlags =  EsOpenWrite | EsOpenCreate;
+	openFlags |= _manager->fileOpenMode(env);
```

Comment:
why don't we merge it in one line?
```suggestion
	int32_t openFlags =  EsOpenWrite | EsOpenCreate | _manager->fileOpenMode(env);
```